src/web/app/components/question-types/question-statistics/mcq-question-statistics.component.ts

```diff
@@ -124,4 +132,31 @@ export class McqQuestionStatisticsComponent
     }
   }
 
+  private sortChoices = (item1: string, item2: string): number => {
+    return this.tableComparatorService.compareLexicographically(item1, item2, SortOrder.ASC);
```

Comment:
You should use `this.tableComparatorService.compare()`, declare the respective enum in `src/web/types/sort-properties.ts` for switch case in the function.